### PR TITLE
Misc: Make an exception for esnext/verbose since they are removed

### PR DESCRIPTION
**Pull request by hzoo**
```js
{
"esprima": true,
    "esprimaOptions": true,
    "verbose": true,
    "esnext": true,
}
```

screen shot 2016-04-16 at 12 05 40 pm

### CODE REVIEW

**Comment by markelog** [comment]
Tests?

File: lib/string-checker.js [hunk 1]
```diff
@@ -158,8 +158,14 @@ StringChecker.prototype = {
             }
         }, this);
 
+        var removed = ['esnext', 'verbose'];
```
Comment (hzoo) [inline lib/string-checker.js]: also esprimaOptions, esprima

**Comment by hzoo** [comment]
Ok that was the wrong place to put that lol

**Comment by zxqfox** [comment]
LGTM

**Pull request merged**